system/handlers/formcontrols/DerivativePicker.cfc

```diff
@@ -0,0 +1,19 @@
+component output=false {
+
+	property name="assetManagerService"  inject="AssetManagerService";
+
+	public string function index( event, rc, prc, args={} ) output=false {
+
+		var derivatives = assetManagerService.listEditorDerivates();
+		args.labels = [];
+		args.values = [];
+
```

Comment:
The `listEditorDerivatives` method should not include a 'none' entry (see comment further down), so we should manually add the 'none' entry for the form control here (but only if there are any derivatives to show).